examples/asr/speech_to_text_eval.py

```diff
@@ -82,7 +82,7 @@ class EvaluationConfig(transcribe_speech.TranscriptionConfig):
     only_score_manifest: bool = False
 
     text_processing: Optional[TextProcessingConfig] = TextProcessingConfig(
-        punctuation_marks=".,?", separate_punctuation=True, do_lowercase=False, rm_punctuation=False,
+        punctuation_marks=".,?", separate_punctuation=False, do_lowercase=False, rm_punctuation=False,
```

Comment:
Do our current PC models on NGC follow the separate_punctuation=False?
The default should work with our current PC models.

Reply:
Yes, Som made a temporary regexp fix to follow this. So, assume that bought target and predicted punctuation is not separated from words.